webpack/assets/javascripts/react_app/common/urlHelpers.js

```diff
@@ -69,3 +70,21 @@ export const changeQuery = (newQuery, navigateTo, uri = getURI()) => {
   if (navigateTo) navigateTo(uri.toString());
   else window.Turbolinks.visit(uri.toString());
 };
+
+export const documentationURL = (section = '', options = {}) => {
+  const shortVersion = packageJson.version
+    .split('.')
+    .slice(0, 2)
+    .join('.');
+  const rootUrl =
+    options.rootUrl ||
+    `https://theforeman.org/manuals/${shortVersion}/index.html#`;
+  if (!section)
+    return `https://theforeman.org/documentation.html#${shortVersion}`;
```

Comment:
This will likely break the theming done for Satellite (which uses different urls for docs)

Reply:
I haven't seen a `documentation_button` that uses a different url than this one in the erb files, how is this different?

Reply:
In downstream we have a branding plugin. Also note that the page you're redirecting to is actually deprecated and redirects to the version specific quickstart guide. Are you sure that's really a useful page?

Reply:
Removed the deprecated link(should it be also removed from ruby?)